web/routes/user-services.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Illuminate\Foundation\Auth\EmailVerificationRequest;
+use Illuminate\Support\Facades\Route;
+use Wikijump\Http\Controllers\AccountController;
+use Wikijump\Http\Controllers\AuthController;
+use Wikijump\Models\User;
+
+// TODO: two factor
+
+// Guest routes
+Route::middleware('guest')->group(function () {
+    Route::view('/login', 'next.auth.login')->name('login');
+
+    Route::view('/register', 'next.auth.register')->name('register');
+
+    // Passsword reset routes
+
+    Route::view('/forgot-password', 'next.auth.forgot-password')->name(
+        'password.request',
+    );
+
+    Route::view('/reset-password/{token}', 'next.auth.reset-password')->name(
+        'password.reset',
+    );
+
+    Route::post('/reset-password/{token}', [
+        AccountController::class,
+        'handlePasswordRecoveryUpdate',
+    ])->name('password.update');
+});
+
+// Auth (unverified) routes
+Route::middleware('auth')->group(function () {
+    // Email verification routes
+
+    Route::get('/verify-email', function (User $user) {
+        // don't allow showing the verification notice if the user is already verified
+        if ($user->hasVerifiedEmail()) {
+            return redirect('/');
```

Comment:
Nitpick: In the future we may want this to redirect to the user's profile/settings page?

Reply:
Sure, when we have one lmao